content/en/users/im/dashboard/_index.md

```diff
@@ -0,0 +1,179 @@
+provided by the IM service in case you need more advanced features use the IM
+web interface.
+
+![Landing page](dash_login.png)
+
+### Login
+
+To login the dashbaord the user will use the [EGI Check-in Service](../../check-in).
+Once authenticated, the user will be redirected to the portfolio of available
+TOSCA templates.
+
+![Portfolio page](dash_configure.png)
+
+### Main menu bar
+
+The main menu bar is located on top of the pages:
+
+- The first button "IM Dashboard" enables the user to go to the the portfolio of
+  available TOSCA templates.
+- Second item "Infrastructures" redirects to the list of current user deployed
+  infrastructures.
+- In the "Advanced" item the "Settings" subitem enables to see the some
+  configuration settings as the URL of the IM service or the OIDC issuer.
+- "External Links" show a set of configurables information links (documentation,
+  video tutorials, etc.)
+- Finally on the right top corner appears the "User" menu item. This item shows
+  the full name of the logged user, and an avatar obtained from
+  [Gravatar](https://www.gravatar.com/). In this menu the user can access their
+  "Cloud Credentials" with the cloud providers or logout the application.
+
+### Cloud Crecentials
+
+This page enable the user to specify the credentials to access any cloud
+provider it has access to. In the list the user can edit, delete and
+enable/disable the selected cloud credential.
+
+![List of Cloud Crecentials](dash_cred_list.png)
+
+Editing or adding the credentials will show a modal form where the user has the
+ability to specify all the parameters needed to access each cloud provider
+supported. In particular for [EGI Cloud Compute](../../cloud-compute) sites the
+user only has to select one of the VOs he is member and one of sites that
+supports that VO. These dropdown fields are generated using the information
+available from the sites and the list of VOs the user is member.
+
+![Edit/Add a Crecential](dash_edit_cred.png)
+
+### TOSCA Templates
+
+The list of available TOSCA templates enable the user to select the required
+topology to deploy.Each TOSCA template can be labelled by the TOSCA developer
+with any "tag" that will show a ribbon displayed on the right bottom corner.
+An special "tag" is the elastic one that are used to mark the templates that
+are configured to automatically manage the elasticity of the deployed cluster.
+
+The user have to click on the "Configure" button to set the input values of the
+TOSCA template and also to select the VO, Site and Image to deploy the
+infrastructure.
+
+Initially the user can set a name to describe the infrastructure to be deployed.
+It will make easier to list infrastructures. In the firsts tabs the user can
+introduce the set of input values of the toplogy. By default there is only one
+tab called "Input Values", but the TOSCA developer can add/rename them to make
+easier the input values selection.
```

Comment:
```suggestion
It will simplify identifying infrastructures. In the firsts tabs, the user can
introduce the set of input values of the topology. By default there is only one
tab called "Input Values", but the TOSCA developer can add or rename them to
simplify the selection of input values.
```